File: python/7 kyu/273.py
# https://www.codewars.com/kata/54ba84be607a92aa900000f1/train/python

# Passed

def is_isogram(string):
    whether_isogram = len(set(string.lower())) == len(string.lower())
    return whether_isogram

output = is_isogram("Dermatoglyphics")
print(output)

File: nodejs/7 kyu/273.js
// https://www.codewars.com/kata/5809c661f15835266900010a/train/javascript

// Passed

function doubleEveryOther(a) {
  let result = a.map((val, idx) => (idx % 2 === 1 ? val * 2 : val));
  return result;
}

const output = doubleEveryOther([1, 2, 3, 4]);
console.log(output);